src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -374,7 +374,7 @@ public void ILLink_verify_analysis_warnings_hello_world_app(string targetFramewo
         }
 
         [Theory]
-        [InlineData("net5.0")]
+        [InlineData("net6.0")]
```

Comment:
Nit: Maybe add the net5.0 version as well and validate that it will NOT change the default - just to be sure we don't "break" existing apps.